crates/sui-types/src/signature.rs

```diff
@@ -32,6 +32,7 @@ pub struct VerifyParams {
     pub zk_login_env: ZkLoginEnv,
     pub verify_legacy_zklogin_address: bool,
     pub accept_zklogin_in_multisig: bool,
+    pub zklogin_max_epoch_upper_bound: Option<EpochId>,
```

Comment:
Should this be Option<u64>? This is not the actual epoch, but just number of epochs of validity? May also want to consider renaming some of these because it's hard to tell which one is actual epoch ID and which one represents a delta (i.e. max number of valid epochs)